### ISSUE TITLE
Fix tests which fail when run in parallel 

### ISSUE THREAD

**Issue by jaredpar** [dotnet/roslyn#321]
The xUnit 2.0 flavor of runners will run tests in parallel and several of our tests fail in that mode.  Disabling for now so we can get to a clean run internally.

**Comment by davkean** [comment]
@jaredpar Is this relevant anymore?

**Comment by jaredpar** [comment]
@davkean yes. Ideally all of our tests would be able to run in parallel as it would give us the best options for test throughput.

**Comment by davkean** [comment]
Can you clarify what work this is representing? I can't tell from it, what the actual action is.

**Comment by jaredpar** [comment]
The work here is essentially changing this line in `app.config` files to enable parallelization.  

https://github.com/dotnet/roslyn/blob/master/src/EditorFeatures/Test/app.config#L6

**Comment by davkean** [comment]
Ah, thanks!

**Comment by davkean** [comment]
I'm guessing your changes, we're just going to won't fix this?

**Comment by jaredpar** [comment]
This should be closed.  It was left open by mistake.  Our tests don't run in parallel within assemblies now.
